xrdp/xrdp_mm.c

```diff
@@ -1030,27 +1030,154 @@ dynamic_monitor_data_first(intptr_t id, int chan_id, char *data, int bytes,
     return 0;
 }
 
+/******************************************************************************/
+static int
+process_dynamic_monitor_description(struct xrdp_wm *wm,
+                                    struct display_size_description *description)
+{
+    int error = 0;
+    struct xrdp_mm *mm = wm->mm;
+    struct xrdp_mod *module = mm->mod;
+
+    LOG_DEVEL(LOG_LEVEL_TRACE, "process_dynamic_monitor_description:");
+
+    if (wm->client_info->suppress_output == 1)
+    {
+        LOG(LOG_LEVEL_ERROR,
+            "\tprocess_dynamic_monitor_description:"
```

Comment:
I'm curious as to why you've used a tab `\t` at the start of a lot of your log messages(?)

Reply:
The idea was to have something like this:

```
process_dynamic_monitor_description:
    process_dynamic_monitor_description: Foobar foobar foobar
```

Anyway, I recognize this is not standard practice throughout XRDP, so I've removed it.

Reply:
I'm happy with that personally, provided the log levels are the same.  Thanks for the explanation.